File: python/problem_5.py
def check_div(input):
    for i in [11, 13, 14, 16, 17, 18, 19, 20]:
        if input % i != 0:
            return False
    return True


def smallest_multiple():
    number = 20
    while True:
        number += 20
        if (check_div(number)):
            return number


answer = 232792560
print(True if smallest_multiple() == answer else False)

File: php/problem_5.php
<?php
function check_div($input)
{
  foreach (array(11, 13, 14, 16, 17, 18, 19, 20) as $i) {
    if ($input % $i != 0) {
      return False;
    }
  }
  return True;
}

function smallest_multiple($number = 20)
{
  while (true) {
    $number += 20;
    if (check_div($number)) {
      return $number;
    }
  }
}

$answer = 232792560;
echo smallest_multiple() == $answer ? "true" : "false";
